doc/api/process.md

```diff
@@ -630,7 +630,17 @@ process.
 ```js
 console.log(`Current directory: ${process.cwd()}`);
 ```
+## process.debugPort
+<!-- YAML
+added: v0.7.2
+-->
+* {number}
 
+A port number for the debugger to listen to.
```

Comment:
Maybe rephrase as "The port used by Node's debugger when it is enabled." or something similar.

Reply:
Alright